scripts/template_upload_scvmm.py

```diff
@@ -171,58 +173,56 @@ def run(**kwargs):
         if new_template_name is None:
             new_template_name = os.path.basename(url)[:-4]
 
-        print("SCVMM:{} started template {} upload".format(provider, new_template_name))
-        print("SCVMM:{} Make Template out of the VHD {}".format(provider, new_template_name))
+        logger.info("SCVMM:{} Make Template out of the VHD {}".format(provider, new_template_name))
 
         # use_library is either user input or we use the cfme_data value
-        use_library = kwargs.get('library', None)
-        if use_library is None:
-            use_library = mgmt_sys['template_upload'].get('library', None) + "\\VHDS\\"
+        library = kwargs.get('library', None)
+        if library is None:
+            library = mgmt_sys['template_upload'].get('library', None) + "\\VHDS\\"
```

Comment:
Consider rolling this into the default arg for `kwargs.get` above instead of having a separate check.